src/main/kotlin/uk/gov/justice/digital/hmpps/courthearingeventreceiver/model/Defendant.kt

```diff
@@ -40,4 +40,7 @@ data class Defendant(
 
   @JsonProperty("croNumber")
   val croNumber: String?,
+
+  @JsonProperty("isYouth")
+  val isYouth: Boolean,
```

Comment:
It is probably not too important, but using `Boolean` as opposed to `boolean` introduces the possibility that the value might be `null` 
`boolean` will default it to false if it is not present.

Reply:
If it is always present obviously this does not matter. I am not certain how Jackson handles nulls downstream, might be worth an Integration test for this to make sure we understand the behaviour. Or alternatively just going with the primitive `boolean`

Reply:
Updated to primitive `boolean`

Reply:
Ignore this, totally forgot this is a Kotlin project where the tri-state Boolean has been abolished - see https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-boolean/

Reply:
Sorry